### PR TITLE
build iris 3.0.0rc0 for label/rc_iris

### PR DESCRIPTION
**Pull request by bjlittle**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bjlittle** [comment]
@conda-forge-admin, please rerender

**Comment by bjlittle** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ test:
 
 about:
   home: http://www.scitools.org.uk/iris
-  license: LGPL-3.0
+  license: LGPL-3.0-only
```
Comment (ocefpaf) [inline recipe/meta.yaml:40]: I'm not a lawyer and I don't understanding anything about licenses but shouldn't this be or-later?

https://github.com/SciTools/iris/blob/master/COPYING#L584-L587

Comment (bjlittle) [reply recipe/meta.yaml:40]: I really don't know, it's always confused me, but then again I am rather dense...
I'll change it to `-or-later` what the heck...

Comment (ocefpaf) [reply recipe/meta.yaml:40]: I'm never really certain about this as well :smile:
If you want to be conservative, and leave `-only` until we figure this out, that would be fine by me.

**Comment by bjlittle** [comment]
@conda-forge-admin, please rerender

**Pull request merged**